src/nickel.rs

```diff
@@ -273,45 +247,64 @@ mod ssl {
     /// extern crate hyper;
     ///
     /// use nickel::Nickel;
+    /// # #[cfg(feature = "ssl")]
     /// use hyper::net::Openssl;
     ///
+    /// # #[cfg(feature = "ssl")]
     /// # fn main() {
     /// let server = Nickel::new();
     /// let ssl = Openssl::with_cert_and_key("foo.crt", "key.pem").unwrap();
     /// server.listen_https("127.0.0.1:6767", ssl);
     /// # }
+    /// # #[cfg(not(feature = "ssl"))]
+    /// # fn main() {}
```

Comment:
Is there an example we can give that works without hyper's `ssl` feature? It seems like our own ssl feature is only for this doctest and the ssl example now. That seems like it could cause confusion, would renaming our feature to `openssl` be accurate in your opinion? 

I guess this is related to your work on other ssl implementations that aren't openssl? Should we be recommending those to users now? (Thanks for the work on those crates btw!) 

(This also fails on Travis due to unused imports: `nickel::Nickel`)